raiden/network/rpc/smartcontract_proxy.py

```diff
@@ -40,21 +40,19 @@ def _check_function_name_and_kargs(self, function_name, kargs):
                 ', '.join(invalid_args),
             ))
 
-    def transact(self, function_name, *args, **kargs):
+    def transact(self, function_name: str, *args, **kargs):
         self._check_function_name_and_kargs(function_name, kargs)
 
         data = self.translator.encode_function_call(function_name, args)
-        txhash = self.transaction_function(
+        return self.transaction_function(
```

Comment:
These small changes shoud be avoided IMO. It's not adding any value, the code is not easier to read nor mantain because of this, and it makes it harder to known the original author of some code.

I would prefer to only make these changes once we agree on a coding standard and then the whole codebase is changed to be consistent with it.